MovieDB/MovieDB/Scenes/Movie Detail/MovieDetailCoordinator.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+
+protocol MovieDetailCoordinatorType {
+    func finish()
+}
+
+final class MovieDetailCoordinator: Coordinator, MovieDetailCoordinatorType {
```

Comment:
As above idea, MovieDetailCoordinator conform only MovieDetailCoordinatorType, not conform Coordinator directly